src/rebar_state.erl

```diff
@@ -289,7 +289,13 @@ apply_profiles(State, Profile) when not is_list(Profile) ->
 apply_profiles(State, [default]) ->
     State;
 apply_profiles(State=#state_t{default = Defaults, current_profiles=CurrentProfiles}, Profiles) ->
-    AppliedProfiles = deduplicate(CurrentProfiles ++ Profiles),
+    AppliedProfiles = case Profiles of
+                          [global | _] ->
```

Comment:
Right, I'll add a comment. But yes, it doesn't clash because it is specially added as the first/only profile when rebar3 installs global plugins.